deploy/kubernetes/base/controller.yaml

```diff
@@ -21,6 +21,30 @@ spec:
       tolerations:
         - key: CriticalAddonsOnly
           operator: Exists
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: "beta.kubernetes.io/os"
+                operator: In
+                values:
+                - linux
+              - key: "beta.kubernetes.io/arch"
+                operator: In
+                values:
+                - amd64
```

Comment:
Why not use `nodeSelector` ?

Reply:
The main manifests for multi architecture that are targeting `arm64` and `amd64` are using affinity, and I just went for it to keep the same pattern, as my cluster to keep consistent with the community usage.

If required I can change to `nodeSelector` and test it.